contrib/ci-workers/files/buildbot-worker.service.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Buildbot worker
+Wants=network.target
+After=network.target
+
+[Service]
+Type=forking
+PIDFile=/home/{{ buildbot_worker_user }}/{{ buildbot_worker_name }}/twistd.pid
+WorkingDirectory=/home/{{ buildbot_worker_user }}
+ExecStart=/usr/local/bin/buildbot-worker start {{ buildbot_worker_name }}
+ExecReload=/usr/local/bin/buildbot-worker restart {{ buildbot_worker_name }}
+ExecStop=/usr/local/bin/buildbot-worker stop {{ buildbot_worker_name }}
```

Comment:
These are in `/usr/local/bin` because the `pip` step uses the global env instead of a virtualenv. We could switch to using a virtualenv if desired, though I don't see much benefit given that these workers will generally be single-purpose (but then, perhaps community-contributed workers won't be...)